avm/res/web/serverfarm/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -56,8 +56,8 @@ module testDeployment '../../../main.bicep' = [
       name: '${namePrefix}${serviceShort}001'
       location: enforcedLocation
       skuName: 'P1v3'
-      skuCapacity: 3
-      zoneRedundant: true
+      skuCapacity: 2
+      zoneRedundant: false
```

Comment:
@tsc-buddy would this configuration work in the target testing subscription? I understood that the whole point was to change the sku back to an available one, but maybe I'm missing something. 
@AlexanderSehr could you also give a look at if this can fix #1396 as well?

Reply:
@AlexanderSehr OK my bad. I understood it was sku related. 

Can we have an issue explaining what we need to try long term to replace this workaround? Otherwise I'm not sure we'll ever remember 🙂

Reply:
@AlexanderSehr is that old one still around? I think it was to do with pipeline failures in general. If so we can keep that or I can make a separate one to track this issue specifically.

Reply:
@eriqua I am hopeful that it will. I've tested it in two environments, a MSDN sub and an internal FTE sub successfully. In US and Asia regions...finger crossed. :)

Reply:
If I recall correctly @eriqua, it's a sku + zonal issue.
In order to deploy zonal, you must use a Premium SKU. But as soon as you do, it doesn't work in our test subscription anymore because (that's what support said) _"It's a preview feature and not available by default"_ which of course is wrong. Preview is mentioned nowhere and it seems to work in all other subscriptions but the AVM & CARML one.

So the fix we agreed upon in the meantime was to disable the zonal test to unblock further PRs.

@tsc-buddy, which 'old one' are you referring to?

Reply:
@AlexanderSehr it was an old pipeline failure issue from april. We can make a new one. Is there anything else i can do on this PR to test it in the AVM environment prior to moving ahead further?

Reply:
@tsc-buddy the `[failed pipeline]` issue will be automatically closed once the pipeline gets green, so a new one is needed.

Once the issue is created you can resolve this comment yourself.

Thanks again for the effort put on this one!

Reply:
I think the suggested change should address the issue @tsc-buddy 💪 Nothing to add from my side

Reply:
issue logged #2303 . Thanks team, will resolve this conversation.